spec/requests/users_spec.rb

```diff
@@ -63,19 +63,26 @@
     end
   end
 
-  describe 'GET /edit' do
-    pending 'not implemented yet'
+  describe 'GET /users/:id/edit' do
     it 'returns http success' do
-      get '/users/edit'
+      user = create(:user)
+      get users_id_edit_path(user)
```

Comment:
Eu acho que tu pode fazer algo como:

```suggestion
      get "/users/#{user.id}/edit"
```